code/modules/exploration_crew/exploration_vendor.dm

```diff
@@ -14,9 +14,6 @@ GLOBAL_VAR_INIT(exploration_points, 0)
 		new /datum/data/vendor_equipment("10 Marker Beacons",			/obj/item/stack/marker_beacon/ten,									300),
 		new /datum/data/vendor_equipment("30 Marker Beacons",			/obj/item/stack/marker_beacon/thirty,								500),
 		new /datum/data/vendor_equipment("Survival Medipen",			/obj/item/reagent_containers/hypospray/medipen/survival,			2000),
-		new /datum/data/vendor_equipment("Brute Healing Kit",			/obj/item/storage/firstaid/brute,									3000),
-		new /datum/data/vendor_equipment("Burn Healing Kit",			/obj/item/storage/firstaid/fire,									3000),
-		new /datum/data/vendor_equipment("Advanced Healing Kit",		/obj/item/storage/firstaid/advanced,								5000),
```

Comment:
exploration crews are already limited to healing themselves as their points are not that very rich as much as miner's. I believe you don't have to touch these.

Reply:
It was 100% intentional.

> Any role being 100% self-reliant is bad.